docs/widgets/media-server.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: 🎞️ Media Server
+tags:
+  - Widgets
+  - Integration
+  - Configuration
+  - Media Server
+  - Plex
+  - Jellyfin
+  - Sessions
+  - API
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The media server widget enables you to watch and view current active sessions over all your media servers.
+It also displays useful information, that you can use when providing support to users or debugging a problem.
+
+---
+
+## Adding the widget
+Please check out our documentation on [how to add a widget](/docs/introduction/after-the-installation#adding-widgets).
+
+---
+
+## Common pitfalls
+Is the widget not working? Please try out the following tips and tricks to possibly resolve your issue:
+
+<Tabs>
+  <TabItem value="jellyfin" label="Jellyfin" default>
+
+
+- Use a URL that does not terminate on a signle ``/`` or something like ``/web`` or ``index.html``. This may break the integration. Only use the basename of your server
+- Ensure that your container has access to the media server. It's recommended that you make a ping or telnet
+- Use valid credentials, that have been created for Homarr in the Jellyfin administration dashboard.
+- Permit access to the required libraries and grant the required permissions
+
+
+  </TabItem>
+  <TabItem value="plex" label="Plex">
+
+
+- Use a URL that does not terminate on a signle ``/`` or something like ``/web`` or ``index.html``. This may break the integration. Only use the basename of your server
+- Ensure that your container has access to the media server. It's recommended that you make a ping or telnet
+- Use a token, that has been created from your Plex. You can retrieve the token using this official guide: https://support.plex.tv/articles/204059436-finding-an-authentication-token-x-plex-token/
+  - Use a token that has not expired
+  - No token visible? Try logging out and in again
+- Do not use the cloud or remote URLs. The integration will only work for local instances. Do not worry, as Homarr will act as a primitve reverse proxy when you're outside the network of Plex
+- Permit access to the required libraries and grant the required permissions
```

Comment:
```suggestion
- Use a URL that does not terminate on a single ``/`` or something like ``/web`` or ``index.html``. This may break the integration. Only use the main domain of your server.
- Make sure that your container has access to the media server. Try making a ping to it.
- Use a token, that has been created from your Plex. You can retrieve the token using this official guide: https://support.plex.tv/articles/204059436-finding-an-authentication-token-x-plex-token/
  - Make sure the token isn't expired.
  - No token visible? Try logging out and in again.
- Do not use the cloud or remote URLs. The integration will only work for local instances. Do not worry, as Homarr will act as a primitve reverse proxy when you're outside the network of Plex.
- Permit access to the required libraries and grant the required permissions.
```